Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -828,15 +828,14 @@ public void chooseDoubleTapButton()
   private void adjustScale()
   {
     LayoutInflater inflater = LayoutInflater.from(this);
-    View view = inflater.inflate(R.layout.dialog_seekbar, null);
+    View view = inflater.inflate(R.layout.dialog_input_adjust, null);
 
-    final SeekBar seekbar = view.findViewById(R.id.seekbar);
-    final TextView value = view.findViewById(R.id.text_value);
-    final TextView units = view.findViewById(R.id.text_units);
+    final SeekBar Scaleseekbar = view.findViewById(R.id.input_scale_seekbar);
```

Comment:
Dolphin's variable naming style (for Java) is `scaleSeekbar`, not `Scaleseekbar`.